bin/pasta_form_patchwork_relations.py

```diff
@@ -0,0 +1,62 @@
+import sys
+import os
+
+from logging import getLogger
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+from pypasta import *
+
+log = getLogger(__name__[-15:])
+
+
+def form_patchwork_relations(config, prog, argv):
+    parser = argparse.ArgumentParser(prog=prog,
+                                     description='Form patch groups using Patchwork IDs instead of Message IDs')
+    parser.add_argument('-infile', metavar='infile', default=None,
+                        help='The patch groups file to be used as input'
+                             'If not specified will default to value of PATCH_GROUPS config variable')
+
+    args = parser.parse_args(argv)
+
+    if config.mode != config.Mode.MBOX:
+        log.error('Only works in Mbox mode!')
+        return -1
+
+    config.repo.register_mbox(config)
+    mbox = config.repo.mbox
+
+    if args.infile is None:
+        f_clustering = config.f_clustering
+    else:
+        f_clustering = args.infile
+
+    f_result = 'patch-groups-patchwork'
```

Comment:
This should be inside resources/project/resources. You can add the filename to the default option of the argument parser.

Reply:
I guess we should also look for the `infile` in /resources/project/resources to be consistent then

Reply:
Otherwise what if for both infile and outfile the user can define whatever path he wants explicitly. If he doesnt define either of those two, we pick up the appropriate default from /resources/project/resources

Reply:
Yes, we can make it an optional argument. But I'd say that it probably won't happen that the user wants to specify a specific patch-groups file other than the one in resources/...

Reply:
Yup, was thinking in terms of enforcing mechanism and not policy :)